lib/app.php

```diff
@@ -39,7 +39,7 @@ public static function getFeed($helper = false, $config = false){
 			$config = \OC::$server->getConfig();
 		}
 		$updater = new \OC\Updater($helper, $config);
-		$data = $updater->check('https://apps.owncloud.com/updater.php');
+		$data = $updater->check('https://updates.owncloud.com/server/');
```

Comment:
Is there a reason why we specify any URL? Without any it will fallback to the default specified one.